### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by abzcoding**
Removed dead configurations, added better `user_plugins` suggestions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,12 +52,12 @@ O.timeoutlen = 100
 O.leader_key = ' '
 
 -- After changing plugin config it is recommended to run :PackerCompile
-O.plugin.hop.active = true
 O.plugin.colorizer.active = true
+O.plugin.dashboard.active = true
+O.plugin.floatterm.active = true
+O.plugin.symbols_outline = true
```
Comment (rebuilt) [inline README.md:58]: This should be singular

```
 O.plugin.symbol_outline = true
```

**Review by rebuilt — CHANGES_REQUESTED** [review]
Just correct symbol_outline

**Comment by rebuilt** [comment]
Thanks for updating the README.  I can't keep up with all the changes :)

**Comment by abzcoding** [comment]
> Just correct symbol_outline

nice catch 👀 , fixed it

**Pull request merged**